pkg/dogstatsd/listeners/packet_buffer_test.go

```diff
@@ -0,0 +1,113 @@
+package listeners
```

Comment:
What happens if we receive a datagram containing `\n`, the rest of the pipeline will consider it as multiple metrics? Should we consider adding some unit tests with that?